### DISCUSSION TITLE
How to Get Current Seed ?

### DISCUSSION THREAD

**Discussion by KumaPlayWithShader** [comfyanonymous/ComfyUI#1504] [General]
The seed will be change to new number when we start a queue. 
I want to find a simple eazy way to reset the seed to old one,
and don't need to save and load  from a png. 
and dont want to load the previous queue. Sometime  I have chage the workflow .  
Load queue will dicard the change.....

**Comment by ltdrdata** [comment]
1. You can roll back workflow to old one on history in main menu.
2. I recommend incr/decr instead of randomize.

**Reply by 2blackbar** [reply]
this is not a solution, solution is updating image preview node with text window that contains prompt info and seed

**Reply by ltdrdata** [reply]
One tricky way is to open a new tab and navigate the history there, allowing you to inspect the values of the existing artifacts while preserving the workflow of the current tab. It's advisable to close the tab immediately after doing this. If you keep multiple tabs open, it's uncertain which workflow will be restored when refreshing, so caution is advised.

The method mentioned above is a solution within the current default ComfyUI implementation. Some third-party seed nodes offer a `control_before_generate` approach instead of `control_after_generate`.
